sycl/doc/extensions/experimental/sycl_ext_oneapi_graph.asciidoc

```diff
@@ -313,6 +313,15 @@ enum class graph_support_level {
   emulated
 };
 
+enum class graph_debug_print_flag {
+  none = 0x00,
+  kernel_args = 0x01,        // adds the printing of the kernel args 
+                             // where applicable
+  memory_access_desc = 0x02  // adds the printing of information related to 
+                             // the memory access: source address, 
+                             // destination address, length, etc.
```

Comment:
I can't find any examples in the main sycl spec of `enum class` definitions specifying values, not sure if it's problematic to break that precedent.